app/services/content_integrity.rb

```diff
@@ -164,7 +164,7 @@ def video?
 
   # @return [Boolean]
   def question_answers?
-    mod.questions.all? { |question| question.answer.valid? && question.feedback_question? }
+    mod.questions.all? { |question| question.answer.valid? || question.feedback_question? }
```

Comment:
@jack-coggin @martikat the only questions we need to ensure have answer options that are present should be factual questions. Alternatively the "validity" check in this predicate needs to do more, rather than just negating it here.